apache/geode
# GEODE-5568: Increase wait in testCacheOpAfterQueryCancel

Fix intermittently-failing test:
QueryMonitorDUnitTest.testCacheOpAfterQueryCancel()
by increasing timeout to account for GC pauses.

Also did some test refactoring cleanup:

 * clean up DefaultQuery.TestHook interface

 * remove dead code

 * improve test comments

 * remove spurious TODOs

 * camelCase some statics that were SCREAMING_CASE

 * fixed typos in test method names

Co-Authored-By: Bill Burcham <bburcham@pivotal.io>
Co-Authored-By: Galen O'Sullivan <gosullivan@pivotal.io>

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:






### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.
